.travis.yml

```diff
@@ -3,16 +3,15 @@ language:
 - node_js
 php:
 - 7.1
-node_js:
```

Comment:
You might need to update lines 1-3 to 
```
dist: xenial
language: ruby
```

if the Travis builds fail with the updated node version. I'm just not sure if the php version affects this.